molecule/pki/host_vars/tls-ats-instance.yaml

```diff
@@ -6,7 +6,7 @@ known_urls:
 # We are forced to use community now otherwise other transformers may conflict with mtls config
 # Remove when mTLS is implemented for SFS and trouter as well.
 acs:
-  artifact_name: alfresco-content-services-community-distribution
-  edition: Community
+  artifact_name: alfresco-content-services-distribution
+  edition: Enterprise
```

Comment:
if we are back to enterprise you should not need anymore this override

Reply:
And that was even written in the comment above 🤦